trust_stores_observatory/store_fetcher/java_fetcher.py

```diff
@@ -93,7 +93,7 @@ def fetch(
         request = Request(
             final_url,
             # Cookie set when 'Accept License Agreement' is selected
-            headers={'Cookie': 'oraclelicense=accept-securebackup-cookie'}
+            headers={'Cookie': 'oraclelicense=accept-securebackup-cookie', 'User-Agent': 'Mozilla/5.0'}
```

Comment:
Should this also mention the project name?

Reply:
That's a good question... I'm not sure. Maybe @nabla-c0d3 knows? 

Also, I'm losing confidence in this and if it this is actually fixing anything.  I made another PR and the build is passing. What if it's random?

Reply:
No certificates added or removed!  When Travis ran an update of the stores, the build failed because a request inside the Java store ( I don't remember which one... either to download the file or to find the link to download the file ) returned a 403 forbidden of doom. I thought it might be because of urllib's user agent. Evidently that is not the case since the build is passing right now 🤔... It just randomly returns an error.